src/lib.rs

```diff
@@ -12,16 +12,48 @@ use bdk::database::{AnyDatabaseConfig, ConfigurableDatabase};
 use bdk::keys::bip39::{Language, Mnemonic, WordCount};
 use bdk::keys::{DerivableKey, ExtendedKey, GeneratableKey, GeneratedKey};
 use bdk::miniscript::BareCtx;
-use bdk::wallet::AddressIndex;
 use bdk::{BlockTime, Error, FeeRate, SignOptions, Wallet as BdkWallet};
-use std::convert::TryFrom;
+use std::convert::{TryFrom, From};
 use std::str::FromStr;
 use std::sync::{Arc, Mutex, MutexGuard};
 
 uniffi_macros::include_scaffolding!("bdk");
 
 type BdkError = Error;
 
+// we redefine a simpler version of bdk::wallet::AddressInfo
+// it has an `address` field of type String instead of the `address` field of type `Address`
+// defined in bitcoin::util::address::Address
```

Comment:
Do we need this? I think it's fairly evident that we're shadowing/proxying here. If it's notes for the reviewers, we can add them as comments instead.

Reply:
Yeah agreed I'll remove them.